src/app/loadout-drawer/subclass-drawer/SubclassPlugDrawer.tsx

```diff
@@ -186,10 +180,15 @@ function getPlugsForSubclass(
         const plugSetHash = firstSocket.socketDefinition.reusablePlugSetHash;
 
         if (plugSetHash && profileResponse) {
-          const plugsWithMaxSelectable: PlugsWithMaxSelectable = {
+          const plugSet: PlugSetWithDefaultPlug = {
             plugs: [],
             plugSetHash,
             maxSelectable: socketGroup.length,
+            defaultPlug: defs.InventoryItem.get(
+              firstSocket.socketDefinition.singleInitialItemHash
+            ) as PluggableInventoryItemDefinition,
```

Comment:
Do we need to use a special function to get the default?

Reply:
Yeah I should, its mainly used for armour mods due to the way artificer armour works but it wouldn't hurt to use it here for future proofing.